IS7024/Pages/HollywoodBowl.cshtml

```diff
@@ -23,20 +23,20 @@
             foreach (QuickTypeBowl.Event eventss in events)
             {
                 string date = eventss.Start.Date.ToString();
-                string[] dars = date.Split('/');
-                string day = dars[0];
-                string month = dars[1];
-                string year = dars[2];
+                string[] dateSplit = date.Split('/');
+                string day = dateSplit[0];
+                string month = dateSplit[1];
+                string year = dateSplit[2];
```

Comment:
Changed variable names to provide further clarification

Reply:
Yes... this is easier to read.